userguide/system.rst

```diff
@@ -742,87 +743,153 @@ Three types of SED devices are supported:
 * TCG Enterprise standard for newer enterprise-grade SAS devices
 
 The %brand% middleware implements the security capabilities of
```

Comment:
We should avoid mentioning middleware as a separate component.  As William would say, it's an implementation detail.